### ISSUE TITLE
Tensorflow 2

### ISSUE THREAD

**Issue by zaher88abd** [kelkelcheng/GC-Net-Tensorflow#1]
Hi,

I am trying to build your model again by Tensorflow 2 because I have some issues running the model on my GPU. The only way I found to run the model is by using Docker; however, it is not easy to make it run in realtime with a camera. My question is, can you give me some ideas about the training process you did for your model like, did you used all the FlyingThings dataset and if there some hyperparameter need to tune. 

Thanks in advance,

**Comment by kelkelcheng** [comment]
Hi Mohamad,

I used the training set of the FlyingThings dataset for training. I also make a tfrecord.

For this model, no hyper-parameter are needed to tune. You can use the same learning rate all the way for training.

Let me know if you have further questions :)

Best,
Kelvin

On Jul 6, 2019, at 12:00 AM, Mohamad Zaher Abd Ulmoula <notifications@github.com<mailto:notifications@github.com>> wrote:


Hi,

I am trying to build your model again by Tensorflow 2 because I have some issues running the model on my GPU. The only way I found to run the model is by using Docker; however, it is not easy to make it run in realtime with a camera. My question is, can you give me some ideas about the training process you did for your model like, did you used all the FlyingThings dataset and if there some hyperparameter need to tune.

Thanks in advance,

**Comment by zaher88abd** [comment]
Thank you.
